pkg/proxy/iptables/proxier_test.go

```diff
@@ -1550,11 +1553,69 @@ func Test_endpointsToEndpointsMap(t *testing.T) {
 				{endpoint: "1.1.1.1:22", isLocal: false},
 			},
 		},
+	}, {
+		// Case: should omit IPv6 address in IPv4 mode
```

Comment:
nit: Can we add test case number, e.g. Case[9]: should omit IPv6 address in IPv4 mode?

Reply:
Done.